### ISSUE TITLE
Cannot fetch forms from google drive

### ISSUE THREAD

**Issue by abbyad** [medic/medic-conf#99]
With v1.18.0 (https://github.com/medic/medic-conf/commit/8105cd95bccf4a72b02887698e9677a8f19eaba2) the `fetch-forms-from-google-drive` action now fails. 

```
$ medic-conf fetch-forms-from-google-drive
INFO Processing config in standard for undefined.
INFO Actions:
     - fetch-forms-from-google-drive
INFO Extra args: undefined
INFO Current version: 1.18.0.  Checking NPM for updates…
INFO You are already on the latest version :¬)
INFO Starting action: fetch-forms-from-google-drive…
ERROR TypeError: Cannot read property 'OAuth2' of undefined
    at oauthClient (/usr/lib/node_modules/medic-conf/src/lib/google-auth.js:65:26)
    at module.exports (/usr/lib/node_modules/medic-conf/src/lib/google-auth.js:12:18)
    at module.exports.projectDir (/usr/lib/node_modules/medic-conf/src/fn/fetch-forms-from-google-drive.js:7:10)
    at promiseChain.then.then (/usr/lib/node_modules/medic-conf/src/bin/medic-conf.js:151:45)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:182:7)
```

Tested on v1.18.0 with Ubuntu (using windows subsystem for linux). It works with v1.17.14.

**Comment by alxndrsn** [comment]
Fixed in `1.18.1`.

**Comment by alxndrsn** [comment]
Fixed in `1.18.1`.

**Comment by alxndrsn** [comment]
Fixed in `1.18.2`.
